src/modules/tracker.js

```diff
@@ -1228,6 +1229,401 @@ class Tracker {
+   *         url: www.example.com
+   *         resultCount: 167,
+   *     },
+   * );
+   */
+  trackQuizResultsLoaded(parameters, networkParameters = {}) {
+    // Ensure parameters are provided (required)
+    if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
+      const requestPath = `${this.options.serviceUrl}/v2/behavioral_action/quiz_result_load?`;
+      const { quizId, quizVersionId, url, section = 'Products', resultCount, resultId, resultPage } = parameters;
+      const queryParams = {};
+      const bodyParams = {};
+
+      if (typeof quizId !== 'string') {
+        return new Error('"quizId" is a required parameter of type string');
+      }
+
+      if (typeof quizVersionId !== 'string') {
+        return new Error('"quizVersionId" is a required parameter of type string');
+      }
+
+      if (typeof url !== 'string') {
+        return new Error('"url" is a required parameter of type string');
+      }
+
+      bodyParams.quizId = quizId;
+      bodyParams.quizVersionId = quizVersionId;
+      bodyParams.url = url;
+
+      if (!helpers.isNil(section)) {
+        if (typeof section !== 'string') {
+          return new Error('"section" must be a string');
+        }
+        queryParams.section = section;
+        bodyParams.section = section;
+      }
+
+      if (!helpers.isNil(resultCount)) {
+        if (typeof resultCount !== 'number') {
+          return new Error('"resultCount" must be a number');
+        }
+        bodyParams.resultCount = resultCount;
+      }
+
+      if (!helpers.isNil(resultId)) {
+        if (typeof resultId !== 'string') {
+          return new Error('"resultId" must be a string');
+        }
+        bodyParams.resultId = resultId;
+      }
+
+      if (!helpers.isNil(resultPage)) {
+        if (typeof resultPage !== 'number') {
+          return new Error('"resultPage" must be a number');
+        }
+        bodyParams.resultPage = resultPage;
+      }
+
+      bodyParams.actionClass = 'result_load';
+
+      const requestURL = `${requestPath}${applyParamsAsString(queryParams, this.options)}`;
+      const requestMethod = 'POST';
+      const bodyParamsSnakeCase = toSnakeCaseKeys(bodyParams);
+      const requestBody = applyParams(bodyParamsSnakeCase, { ...this.options, requestMethod });
+
+      this.requests.queue(
+        requestURL,
+        requestMethod,
+        requestBody,
+        networkParameters,
+      );
+      this.requests.send();
+
+      return true;
+    }
+
+    this.requests.send();
+
+    return new Error('parameters are required of type object');
+  }
+
+  /**
+   * Send quiz result click event to API
+   *
+   * @function trackQuizResultClick
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.quizId - Quiz Id
+   * @param {string} parameters.quizVersionId - Quiz version Id
+   * @param {string} [parameters.itemId] - Product item unique identifier
```

Comment:
I see you added logic to check one of itemId or itemName exists. Can we also add a note here saying you need to pass one of these?